.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,36 @@
+name: E2E
+on:
+  schedule:
+    - cron: '14 4 * *  *'
+  workflow_dispatch:
+
+jobs:
+  e2e-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1.7
+      - uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # v4.1.0
+        with:
+          node-version: 20
+      - name: run tests
+        env:
+          OC_LOGIN_TOKEN: ${{ secrets.OC_LOGIN_TOKEN }}
+          OC_SERVER_URL: ${{ secrets.OC_SERVER_URL }}
+          MY_GITHUB_TOKEN: ${{ secrets.MY_GITHUB_TOKEN }}
+          DEVELOPER_HUB_URL: ${{ secrets.DEVELOPER_HUB_URL }}
```

Comment:
how are we setting these ENV

Reply:
as repository secrets I hope, will just need somebody with admin right to this repo

Reply:
The idea is to use a long running cluster, and keep RHDH in the same namespace, specifically so jobs like this one wouldn't need constant updates. Also, serviceaccount tokens have quite long and happy lives.